src/Data/TemplateData.js

```diff
@@ -599,7 +598,7 @@ class TemplateData {
 		}
 
 		debug("getLocalDataPaths(%o): %o", templatePath, paths);
-		return unique(paths).reverse();
+		return [...new Set(paths)].reverse();
```

Comment:
i'm not sure this javascript incantation is necessarily easier to understand than a named function

Reply:
True. I’m all over the place :( I will fix this one.